### DISCUSSION TITLE
Can't ever duplicate tables in the UI

### DISCUSSION THREAD

**Discussion by trevorwrightdev** [supabase/supabase#10985] [Questions]
I really really want to just be able to duplicate a table in Supabase for testing purposes. But every time I click duplicate, I get an SQL error pop up. This is the error: 
<img width="339" alt="image" src="https://user-images.githubusercontent.com/86077703/207725372-d7485b8c-3a3d-4513-bcc9-3f5aa7a8d238.png">
It is funny because that isn't even the table name. It is "MutantKaijuTokens." I've tested it, and the issue is this casing problem. Is there a solution to this or another way I can duplicate a table?

**Comment by GaryAustin1** [comment]
Postgres does not play well with capital letters, in general.  You should avoid them if given a choice.  But the UI could deal with it, but must not in the duplicate code.  So you need to generate an issue (if it does not already exist).
